consensus/hotstuff/vote_collector.go

```diff
@@ -43,11 +54,19 @@ type VoteCollector interface {
 	Status() VoteCollectorStatus
 }
 
-// VerifyingVoteCollector must implement the same interface as BlockSigner, such that when eventhandler
-// asks voter to vote for the block, the voter will ask VoteAggregator(via BlockSigner interface)
-// to sign the block, and VoteAggregator will read the VerifyingVoteCollector from the vote collectors
-// map and produce the vote.
-// Note CachingVoteCollector can't create vote, only VerifyingVoteCollector can
-type VerifyingVoteCollector interface {
```

Comment:
This one is an implementation detail and I have just removed it, we can reintroduce it inside of votecollector/interface.go for instance but I don't really know if there is a need.